config/initializers/date_formats.rb

```diff
@@ -0,0 +1,6 @@
+#Initialise default Gov UK date formats
```

Comment:
It might be worth referencing the style guide in the comment too for some additional context. Also we should say "GOV.UK" :)